crates/rustc_codegen_spirv/src/target.rs

```diff
@@ -84,7 +84,7 @@ impl SpirvTarget {
         o.dll_suffix = ".spv".into();
         o.dynamic_linking = true;
         o.emit_debug_gdb_scripts = false;
-        o.linker_flavor = LinkerFlavor::Ld;
+        o.linker_flavor = LinkerFlavor::Unix(Cc::No);
```

Comment:
Interesting, we should revisit this at some point, because we basically have our own custom linker, and we may want to treat parameters to it as "linker flags" etc.